src/ISA/ISA/JsonTypes/OntologyAnnotation.fs

```diff
@@ -28,6 +28,7 @@ type OntologyAnnotation =
         OntologyAnnotation.create()
 
     /// Returns the name of the ontology as string
+    // TODO: Why is this called Text, while everything else is called string?
```

Comment:
I think in this specific case it was input from Timo. Agree that we should stick to one convention.